From f5a34e771532b8433f307b679500c26af328ba35 Mon Sep 17 00:00:00 2001
From: Changqing Li <changqing.li@windriver.com>
Date: Fri, 17 Apr 2020 15:01:57 +0800
Subject: [PATCH] configure.ac: don't include build/libtool.m4

we delete build/libtool.m4 before do_configure,
we will use libtool.m4 under ACLOCALDIR

Upstream-Status: Inappropriate [oe-specific]

Signed-off-by: Changqing Li <changqing.li@windriver.com>
---
 configure.ac | 1 -
 1 file changed, 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 06c6a27..f85144e 100644
--- a/configure.ac
+++ b/configure.ac
@@ -6,7 +6,6 @@ dnl ----------------------------------------------------------------------------
 m4_include([build/ax_check_compile_flag.m4])
 m4_include([build/ax_func_which_gethostbyname_r.m4])
 m4_include([build/ax_gcc_func_attribute.m4])
-m4_include([build/libtool.m4])
 m4_include([build/php_cxx_compile_stdcxx.m4])
 m4_include([build/php.m4])
 m4_include([build/pkg.m4])
-- 
2.7.4

